reseq_ros2/reseq_ros2/agevar.py

```diff
@@ -58,8 +58,8 @@ def __init__(self):
 
     def remote_callback(self, msg):
         # extract information from ROS Twist message
-        linear_vel = msg.linear.x
-        angular_vel = linear_vel * msg.angular.z
+        linear_vel = msg.linear.y
```

Comment:
We were using the X axes because that's what was used in the document describing the math behind agevar. I don't know if I would change it to Y, even though of course that makes more sense with the remote